app/controllers/project_sboms_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class ProjectSbomsController < ApplicationController
+  helper ProjectsHelper
+
+  before_action :find_project
```

Comment:
There is an function in `ApplicationController` named `set_project_or_fail`. Use it here instead.
Don't check for `if @project` in the `index` action, since `set_project_or_fail` will raise exception when `@project` is not set.